starboard/extension/loader_app_metrics.h

```diff
@@ -38,6 +38,16 @@ typedef enum CrashpadInstallationStatus {
   kMaxValue = kFailedSignalHandlerInstallationFailed
 } CrashpadInstallationStatus;
 
+typedef enum class SlotSelectionStatus {
+  // The enumerators below this point were added in version 3 or later.
+  kUnknown = 0,
+  kCurrentSlot = 1,
```

Comment:
It looks like this is unused so far. Is that intentional?

Reply:
Hey did you mean to resolve this? My question was about the `kCurrentSlot` value being unused, and it still is.

Reply:
We should keep kCurrentSlot in case we need to use it in the future. It's also for the logic completeness of SlotSelectionStatus.